nmi_mysql/nmi_mysql.py

```diff
@@ -134,6 +149,8 @@ def _execute(self, _query, _params, _multi=False):
                         'affected_rows': result.rowcount
                     }
 
+            con.close()
```

Comment:
Actually, the external should be closing the connection when they don't need to use the connection so it is returned to the pool. (ie: they may hold the connection for several queries, then close it when they are done.)

That said, I think this should work fine here, as the pool well then handle the connection connect/close and make sure there's always an active connection for the query.